package.json

```diff
@@ -1,22 +1,51 @@
 {
 	"name": "resolume-arena",
-	"version": "2.0.2",
-	"main": "src/index.js",
-	"type": "module",
+	"main": "dist/index.js",
+	"version": "3.0.0",
+	"description": "Resolume Arena 6 & 7 module for companion",
+	"api_version": "1.0.0",
+	"keywords": [
+		"Video",
+		"Software",
+		"Graphics"
+	],
+	"manufacturer": "Resolume",
+	"product": "Arena",
+	"shortname": "arena",
+	"homepage": "https://github.com/bitfocus/companion-module-resolume-arena#readme",
 	"scripts": {
-		"test": "echo \"Error: no test specified\" && exit 1",
-		"format": "prettier --write ."
+		"dev": "yarn build:watch",
+		"build": "npx rimraf dist && npx -p typescript tsc -p tsconfig.build.json",
+		"build:main": "tsc -p tsconfig.build.json",
+		"build:watch": "tsc -p tsconfig.build.json --watch",
+		"lint": "eslint . --ext .ts",
+		"postinstall": "yarn build",
+		"test": "qunit --require ts-node/register 'test/**/*-test.ts'"
+	},
+	"license": "MIT",
+	"authors": [
+		"Oliver Herman <oliver@monoxane.com>",
+		"Bryan Crotaz <bgcrotaz@hotmail.com>"
+	],
+	"bugs": {
+		"url": "https://github.com/bitfocus/companion-module-resolume-arena/issues"
 	},
 	"repository": {
 		"type": "git",
 		"url": "git+https://github.com/bitfocus/companion-module-resolume-arena.git"
 	},
-	"license": "MIT",
 	"dependencies": {
-		"@companion-module/base": "~1.4"
+		"node-fetch": "^2.0.0",
```

Comment:
The version of nodejs we use has a version of fetch builtin, you may be able to avoid using a library for this now
https://nodejs.org/dist/latest-v18.x/docs/api/globals.html

Reply:
I tried, but does not seem to be working yet, I see fetch is still experimental in v18.19, might want to wait till v21.